WcaOnRails/.gitignore

```diff
@@ -24,7 +24,9 @@ public/assets/**
 app/webpacker/packs/generated_locales/*.js
 
 # Environment configuration
-.env*
```

Comment:
I'd suggest leaving this line as-is and explicitly un-ignore any relevant files like the test env:
```gitignore
.env*
!.env.test
```